src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -381,6 +378,15 @@ protected override void ProcessRecord()
                     ErrorRecord er = new ErrorRecord(ex, "WebCmdletWebResponseException", ErrorCategory.InvalidOperation, request);
                     ThrowTerminatingError(er);
                 }
+                finally
+                {
+                    // We are done using the request (HttpRequestMessage), so make sure that it gets garbage collected.
+                    // This forces to dispose resources that were used to create the request.
+                    if (request != null)
+                    {
+                        request.Dispose();
```

Comment:
>Dispose [](start = 32, length = 7)

does this close file handle used in the request? do  you need to do that explicitly?